resource-timing/resources/iframe-TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script>
+    const url = '{{location[scheme]}}://{{host}}:{{ports[http][1]}}/resource-timing/resources/TAOResponse.py?tao=origin_port_{{ports[http][1]}}';
+    let observe = (list, observer) => {
+        let entry = list.getEntries()[0];
+        if (!entry) {
```

Comment:
This is not needed. The callback will not be called when there are no entries.

Reply:
sure